flocker/node/agents/ebs.py

```diff
@@ -174,7 +174,6 @@ def _blockdevicevolume_from_ebs_volume(ebs_volume):
 
     :return: Input volume in BlockDeviceVolume format.
     """
-    ebs_volume.update()
```

Comment:
If this is an issue, it seems that `_wait_for_volume` could probably return the most up-to-date ebs volume object and code that previously used a kind-of out-of-date object could use that recently-updated one instead.